### ISSUE TITLE
Exception when using build-info-extractor-gradle latest version with 

### ISSUE THREAD

**Issue by aj-swipejobs** [jfrog/build-info#599]
I'm using the  build-info-extractor-gradle plugin(version 4.25.5) and gradle.plugin.com.palantir:gradle-flexversion version 0.9.2 in our projects. And since yesterday all my builds started to fail due to the following error 

```
No signature of method: org.eclipse.jgit.internal.storage.file.FileRepository.getRef() is applicable for argument types: (String) values: [HEAD]
  Possible solutions: getAt(java.lang.String), getFS(), grep(), exactRef(java.lang.String), findRef(java.lang.String), getAllRefs()
```

It looks like when it's paired with `gradle.plugin.com.palantir:gradle-flexversion:0.9.2` this issue starts happening. May be newest plugin using a newer groovy version?

**Versions**
- Gradle Artifactory plugin version: 4.25.5
- Operating system: macOS Monterey
- Gradle version: 4.10.2

**Comment by Or-Geva** [comment]
Thank you @aj-swipejobs, for taking the time to report this issue.
We have released Gradle Artifactory plugin 4.26.1, which includes the fix for this issue.

**Comment by aj-swipejobs** [comment]
Hi @Or-Geva, thanks for fixing this one :). I can confirm it's working on my side at-least :).
